py/picca/data.py

```diff
@@ -12,30 +12,6 @@
 from picca.utils import userprint, unred
 from picca.dla import DLA
 
-
-def get_variance(var_pipe, eta, var_lss, fudge):
```

Comment:
If I recall properly, this was left as a function since it was used in multiple places. I understand removing it and making the computation explicitly in class Forest but, what happened to the other places this was used?

Reply:
Ok, I saw that you modified the other places as well